app/src/main/res/layout/reset_password_by_token_fragment.xml

```diff
@@ -29,6 +29,23 @@
                 android:paddingRight="@dimen/spacing_normal"
                 android:paddingStart="@dimen/spacing_normal">
 
+                <TextView
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:text="@string/check_your_email"
+                    android:layout_gravity="center_horizontal"
+                    android:textColor="@color/color_accent"
+                    android:textSize="@dimen/text_size_larger"
+                    style="@style/Base.TextAppearance.AppCompat.Medium"
+                    android:layout_marginBottom="@dimen/spacing_normal"/>
+
+                <TextView
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:text="@string/check_email_reset_password_msg"
```

Comment:
It appears that this string is displayed even when there is an error in token request and token is not sent to email which should not be the case.